tests/test_lit_server.py

```diff
@@ -562,3 +562,65 @@ async def test_async_litapi():
             resp = await ac.post("/predict", json={"input": 5.0}, timeout=10)
             assert resp.status_code == 200, "Server response should be 200 (OK)"
             assert resp.json()["output"] == 25.0, "output from Identity server must be same as input"
+
+
+class TestSleepAsyncLitAPI(ls.LitAPI):
+    def setup(self, device):
+        self.model = None
+
+    async def decode_request(self, request):
+        return request["input"]
+
+    async def predict(self, x):
+        # sleep x time to simulate a long-running task
+        await asyncio.sleep(x)
```

Comment:
We can have a fixed sleep time here and send concurrent (maybe 10) requests from the client. 

To assert that the server is concurrent and not sequential, we should get the response for all the requests within 4 seconds instead of `num_requests x 4`. 

```suggestion
        await asyncio.sleep(4)
```